evalml/pipelines/pipeline_base.py

```diff
@@ -667,6 +667,7 @@ def new(self, parameters, random_seed=0):
         Returns:
             A new instance of this pipeline with identical components.
         """
+        # --> why is there no setting of the threshold for binary like there is in clone?
         return self.__class__(
```

Comment:
Not really relevant to the rest of the PR, but I noticed that the following check that's in `clone` isn't in this method and wanted to see if anyone knew why:

```python
        if is_binary(self.problem_type):
            clone.threshold = self.threshold
```